jwst/pipeline/calwebb_spec3.cfg

```diff
@@ -0,0 +1,18 @@
+name = "Spec3Pipeline"
+class = "jwst.pipeline.Spec3Pipeline"
+suffix = 'x1d'
+save_results = True
+
+    [steps]
+      [[skymatch]]
+        suffix = 'skymatch'
+      [[outlier_detection]]
+        suffix = 'crj'
```

Comment:
Ok, that would make sense.  `_a3001_crf.fits` is much nicer than what we have currently.